spec/fixer_currency_with_json_spec.rb

```diff
@@ -0,0 +1,143 @@
+require File.expand_path(File.dirname(__FILE__) + '/spec_helper')
+
+require 'money'
+require 'money/bank/fixer_currency'
+
+describe Money::Bank::FixerCurrency do
+  before :each do
+    @bank = Money::Bank::FixerCurrency.new
+    @bank.access_key = '123'
```

Comment:
can this be passed in as a constructor argument for convenience? or is this a limitation of the way it integrates with the money gem?

Reply:
or perhaps set as a class variable once?